src/content/guides/creating-clusters-via-crs-on-aws/index.md

```diff
@@ -221,3 +221,20 @@ There are also specific reference pages for [cluster templating](/reference/kube
 As a result of rendering the CRs ([sample](/reference/kubectl-gs/template-cluster/#example)), a user will get YAML manifests containing valid CRs that can create a tenant cluster and its node pools.
 The resources can then be created by applying the manifest files to the Control Plane, e.g. `kubectl create -f <cluster manifest file>.yaml`.
 Of course, that requires the user to be authorized towards Kubernetes Control Plane API.
+
+## How to delete a cluster using Cluster API
+
+In order to delete a cluster succesfully all the resources belonging to this instance need to be marked for deletion, this includes:
+
+- `Cluster` (API version `cluster.x-k8s.io/v1alpha2`)
```

Comment:
It should be good enough to delete the `Cluster` CR. `cluster-operator` should propagate the deletion to all other CRs. This is also the behaviour of our conformance tests.

Reply:
hmm true, will move to different place and document that they can also only remove the cluster